src/hotspot/share/gc/serial/cardTableRS.cpp

```diff
@@ -26,19 +26,18 @@
 #include "classfile/classLoaderDataGraph.hpp"
 #include "gc/serial/cardTableRS.hpp"
 #include "gc/serial/generation.hpp"
-#include "gc/serial/serialHeap.hpp"
-#include "gc/serial/tenuredGeneration.hpp"
+#include "gc/serial/serialHeap.inline.hpp"
 #include "gc/shared/space.inline.hpp"
 #include "memory/iterator.inline.hpp"
 #include "utilities/align.hpp"
 
-void CardTableRS::younger_refs_in_space_iterate(TenuredSpace* sp,
-                                                OopIterateClosure* cl) {
+void CardTableRS::younger_refs_in_space_iterate(TenuredSpace* sp) {
```

Comment:
The "iterate" string in `younger_refs_in_space_iterate` alluded that this function was written as generic function with the code provided by a passed-in closure. Now that this function is locked down to be a scan function, I think the name should be changed. This applies to the callers as well.